packages/edge/README.md

```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
 Then follow the same document for registering the Edge extension with the Mobile Core.
 Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
 
+
+Initializing Example:
```

Comment:
Let's rename Initializing Example: -> **Initialization example** - same for all files